orttraining/orttraining/python/training/ortmodule/_io.py

```diff
@@ -129,8 +129,20 @@ def _combine_input_buffers_initializers(params, onnx_input_names, input_info, bu
         * Initializers: computed from original PyTorch model parameters
     '''
 
+    def _expand_inputs(current_input, non_none_inputs):
+        if isinstance(current_input, abc.Sequence):
```

Comment:
Do we need to exclude strings here, or is that covered by an earlier `isinstance`?

Reply:
We don't need another ```isinstance``` for ```str``` checking here as it is covered earlier.